style.css

```diff
@@ -0,0 +1,84 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+  html {
+    box-sizing: border-box;
+  
+    
+    font-size: 50%;
+    scroll-behavior: smooth;
+  }
+  @media (max-width: 65em) {
+    html {
+      font-size: 50%;
+    }
+  }
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.8rem;
+    font-weight: 200;
+
+    
```

Comment:
- [ ] Please try to be more professional in your comments and add more solid reasons, also it seems like you are ordering the student, your comment should be clear, polite and professional.

## Your comment

>  Could you please review and remove the unnecessary empty new lines? This will help keep the code more concise and readable.